Makefile

```diff
@@ -7,3 +9,19 @@ run:
 run-grpc:
 	@go run main.go start -g true -r 3939
 
+test-dep:
+	@examples/./server 2>/dev/null & echo $$! > testserver.pid
```

Comment:
If the server is used for tests it should not be inside the examples folder.

Why examples/./server instead of ./examples/server?